### ISSUE TITLE
Property.array(ObjectId)

### ISSUE THREAD

**Issue by Limpidius-Fred** [strongloop/loopback-connector-mongodb#587]
## Steps to reproduce

```
  class MyModel {
    @property.array(String, {
      default: [],
      mongodb: {dataType: 'ObjectID'},
    }) relatedIds: string[];
  }
```

## Current Behavior

relatedIds writes strings[] into MongoDb

## Expected Behavior

relatedIds should write ObjectId[] into MongoDb

## Additional information

This looks like correcting the problem (mongodb.js#36, function ObjectId):
```
  if (typeof id !== 'string') {
    return Array.isArray(id) ? id.map(ObjectID) : id;
  }

```

**Comment by agnes512** [comment]
I think this is fixed in https://github.com/strongloop/loopback-connector-mongodb/pull/582 and released in version 5.3.0. Thanks.

**Comment by agnes512** [comment]
Similar issue: https://github.com/strongloop/loopback-next/issues/5848
